### ISSUE TITLE
Linear instability waves

### ISSUE THREAD

**Issue by JianfengNing** [lu-group/deeponet-fno#17]
Dear Prof. Lu,

For the numerical experiment  "Linear instability waves in high-speed boundary layers",  in the arXiv version, the FNO is unstable to added noise, while in the published version, the FNO is stable to noise.  I also have some numerical experiences that FNO is unstable to noise. Could you please explain why you have found that FNO is stable to noise in your published version? Thank you.

**Comment by lululxvi** [comment]
Yes, FNO could be unstable. But by some tuning, the stability may be improved.

**Comment by profPlum** [comment]
@lululxvi do you remember how you improved the FNO stability?

**Comment by lululxvi** [comment]
I forgot which hyperparameters are important for stability.
